.env.production.example

```
@@ -39,15 +39,20 @@ AUTH_PROVIDER=supabase
+NEXT_PUBLIC_APP_URL=https://incfrog.ai
+
+# Private service-to-service URL the Next.js server uses to reach NestJS on the
+# Docker network. Without it the web container's /api/proxy and /api/auth route
+# handlers fall back to the public URL and loop out through nginx.
+INTERNAL_API_URL=http://api:4000
 
 # Internal URL the API uses to reach itself / web
-WEB_BASE_URL=https://<your-domain>
-APP_BASE_URL=https://<your-domain>
+WEB_BASE_URL=https://incfrog.ai
+APP_BASE_URL=https://incfrog.ai
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_BASE_URL key should go before the WEB_BASE_URL key

(UnorderedKey)